app/components/message_attachments/message_attachment.js

```diff
@@ -97,7 +97,7 @@ export default class MessageAttachment extends PureComponent {
                         navigator={navigator}
                         onPermalinkPress={onPermalinkPress}
                         textStyles={textStyles}
-                        value={attachment.text || attachment.fallback}
+                        value={attachment.text}
```

Comment:
This will cause that the fallback does not get rendered in case there is no text

Reply:
Hey @enahum, I meant to reach out to you earlier.  It looks like these types of items should only fall through the the AttachmentFields component and not AttachmentText. The duplication issue is caused when the AttachmentText passes through the `fallback` field AND the AttachmentFields component shows the data as well.

Reply:
I commented elsewhere on this, but we're not actually supposed to show the `fallback` value anywhere. It's intended to only be used when proper message attachment formatting is not supported such as in a notification.